Mathlib/Algebra/Order/Ring/Lemmas.lean

```diff
@@ -9,110 +9,160 @@ import Mathlib.Algebra.GroupWithZero.Defs
+
+See https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/notation.20for.20positive.20elements
+for a discussion about this notation, and whether to enable it globally (note that the notation is
+currently global but broken, hence actually only works locally).
 -/
 
 
 variable (α : Type*)
 
--- mathport name: «exprα≥0»
-/- Notations for nonnegative and positive elements
-https://
-leanprover.zulipchat.com/#narrow/stream/113488-general/topic/notation.20for.20positive.20elements
--/
-section Abbreviations
-
-variable [Mul α] [Zero α] [Preorder α]
-
 set_option quotPrecheck false in
 /-- Local notation for the nonnegative elements of a type `α`. TODO: actually make local. -/
 notation "α≥0" => { x : α // 0 ≤ x }
 
--- mathport name: «exprα>0»
 set_option quotPrecheck false in
 /-- Local notation for the positive elements of a type `α`. TODO: actually make local. -/
 notation "α>0" => { x : α // 0 < x }
 
-/-- `PosMulMono α` is an abbreviation for `CovariantClass α≥0 α (fun x y ↦ x * y) (≤)`,
-expressing that multiplication by nonnegative elements on the left is monotone. -/
+section Abbreviations
+
+variable [Mul α] [Zero α] [Preorder α]
+
+/-- Typeclass for monotonicity of multiplication by nonnegative elements on the left,
+namely `b₁ ≤ b₂ → a * b₁ ≤ a * b₂` if `0 ≤ a`.
+
+You should usually not use this very granular typeclass directly, but rather a typeclass like
+`OrderedSemiring`. -/
 abbrev PosMulMono : Prop :=
   CovariantClass α≥0 α (fun x y => x * y) (· ≤ ·)
 #align pos_mul_mono PosMulMono
 
-/-- `MulPosMono α` is an abbreviation for `CovariantClass α≥0 α (fun x y ↦ y * x) (≤)`,
-expressing that multiplication by nonnegative elements on the right is monotone. -/
+/-- Typeclass for monotonicity of multiplication by nonnegative elements on the left,
+namely `a₁ ≤ a₂ → a₁ * b ≤ a₂ * b` if `0 ≤ b`.
+
+You should usually not use this very granular typeclass directly, but rather a typeclass like
+`OrderedSemiring`. -/
 abbrev MulPosMono : Prop :=
   CovariantClass α≥0 α (fun x y => y * x) (· ≤ ·)
 #align mul_pos_mono MulPosMono
 
-/-- `PosMulStrictMono α` is an abbreviation for `CovariantClass α>0 α (fun x y ↦ x * y) (<)`,
-expressing that multiplication by positive elements on the left is strictly monotone. -/
+
+/-- Typeclass for strict monotonicity of multiplication by positive elements on the left,
+namely `b₁ < b₂ → a * b₁ < a * b₂` if `0 < a`.
+
+You should usually not use this very granular typeclass directly, but rather a typeclass like
+`StrictOrderedSemiring`. -/
 abbrev PosMulStrictMono : Prop :=
   CovariantClass α>0 α (fun x y => x * y) (· < ·)
 #align pos_mul_strict_mono PosMulStrictMono
 
-/-- `MulPosStrictMono α` is an abbreviation for `CovariantClass α>0 α (fun x y ↦ y * x) (<)`,
-expressing that multiplication by positive elements on the right is strictly monotone. -/
+/-- Typeclass for strict monotonicity of multiplication by positive elements on the left,
+namely `a₁ < a₂ → a₁ * b < a₂ * b` if `0 < b`.
+
```

Comment:
You've exchanged `right` with `left` in this docstring; I assume that's a copy-paste error?

Reply:
Same throughout